src/graphnet/utils/cluster/condor.py

```diff
@@ -0,0 +1,843 @@
+                self.metricsFileInfo.end_time
+            )
+            returnString += "    Time taken in total  : %s\n" % (
+                time.human_time(self.metricsFileInfo.duration)
+            )
+            returnString += "    DAG status           : %s\n" % (
+                interpretDAGStatus(self.metricsFileInfo.dag_status)
+            )
+        else:
+            returnString += "  [Metrics file not parsed]\n"
+
+        return returnString
+
+
+# Subclass containing submit file information
+class DAGManMetricsSubmitFileInfo(object):
+    def __init__(self, dagmanSubmitFilePath):
+        self.parse(dagmanSubmitFilePath)
+
+    def fullyParsed(self):
+        return True  # No conditional parsing for this file at this time
+
+    def parse(self, dagmanSubmitFilePath):
+
+        # Check submit exists, open it, and read it into a string
+        if os.path.exists(dagmanSubmitFilePath):
+            with open(dagmanSubmitFilePath) as submit_file:
+                submit_fileText = submit_file.read()
+
+                # Count the instances of the "JOB" ketword in the string
+                self.numJobs = submit_fileText.count("JOB")
+
+        else:
+            raise Exception(
+                'DAGMan submit file "%s" does not exist' % dagmanSubmitFilePath
+            )
+
+
+# Subclass containing metrics file information
+class DAGManMetricsMetricsFileInfo(object):
+    def __init__(self, metricsJsonFilePath):
+        self.parse(metricsJsonFilePath)
+
+    def fullyParsed(self):
+        return True  # No conditional parsing for this file at this time
+
+    def parse(self, metricsJsonFilePath):
+
+        # Open the JSON metrics file and parse using a json parser
+        if os.path.exists(metricsJsonFilePath):
+            with open(metricsJsonFilePath) as dagmanMetricsFile:
+
+                metricsNode = json.load(dagmanMetricsFile)
+                if metricsNode:
+
+                    # Read individual details from the metrics file
+                    self.numJobs = metricsNode["jobs"]
+                    self.numJobsSucceeded = metricsNode["jobs_succeeded"]
+                    self.numJobsFailed = metricsNode["jobs_failed"]
+                    self.start_time = metricsNode["start_time"]
+                    self.end_time = metricsNode["end_time"]
+                    self.duration = metricsNode["duration"]
+                    # self.jobsCombinedDuration = metricsNode['total_job_time'] #TODO replace once DAGman actually writes this variable, currently always 0
+                    self.dag_status = metricsNode["dag_status"]
+
+                else:
+                    raise Exception(
+                        'DAGMan metrics file "%s" cannot be parsed as a JSON file'
+                        % metricsJsonFilePath
+                    )
+
+        else:
+            raise Exception(
+                'DAGMan metrics file "%s" does not exist' % metricsJsonFilePath
+            )
```

Comment:
I don't see this class referenced elsewhere (apart from one or more places that are not themselves references elsewhere)?
```suggestion
```